lib/pkcs7.js

```diff
@@ -342,16 +342,20 @@ p7.createSignedData = function() {
         if('content' in msg) {
           var content;
           if(msg.content instanceof forge.util.ByteBuffer) {
-            content = msg.content.bytes();
+            content = asn1.create(asn1.Class.UNIVERSAL, asn1.Type.OCTETSTRING, false,
```

Comment:
I believe this would break the design here. The format for the `content` of PKCS#7 messages is dependent on the `contentType`. The current design allows for people to specify a `ByteBuffer` (or binary string of bytes) with DER-encoded content -- that expresses whatever ASN.1 format they desire. This would force the content to be an `asn1.Type.OCTETSTRING`, which is undesirable.